mappings/net/minecraft/entity/ai/goal/AttackWithOwnerGoal.mapping

```diff
@@ -1,4 +1,6 @@
 CLASS aov net/minecraft/entity/ai/goal/AttackWithOwnerGoal
-	FIELD a owner Lajc;
+	FIELD a tamedEntity Lajc;
```

Comment:
what's wrong with owner? it indicates the goal's owner.

Reply:
Owner is very confusing here, because we have both the owner of the goal, and the owner of the goal's owner. We probably shouldn't name them `owner` and `ownerOwner`.

Reply:
Let's choose another universal word for the entity that goals apply to... applicant?

Reply:
I suggest `entity`, `cat`, `zombie`, etc., see https://github.com/FabricMC/yarn/issues/661

Reply:
Ah, `actor`. How about that? We should change all `owner` to `actor`, which is sensible and not confusing any more.

Reply:
But it will still be inconsistent because some Goals are extended by others, which then store a less general `actor`. If the super class already uses actor, not possible anymore.
I suggest to use `actor` in every class that extends `Goal` directly and `<entity>Actor` when `actor` is not possible.